image.yaml

```diff
@@ -0,0 +1,29 @@
+schema_version: 1
+name: "jboss-dataservices/datagrid-online-services"
+description: "Red Hat JBoss Data Grid Services"
+version: "1.0"
+from: "jboss-datagrid-7/datagrid71-openshift:1.0"
+labels:
+    - name: "io.k8s.description"
+      value: "Provides a scalable in-memory caching solution for fast access to large volumes of data."
+    - name: "io.k8s.display-name"
+      value: "JBoss Data Grid Online Services"
+    - name: "io.openshift.tags"
+      value: "datagrid,java,jboss,xpaas"
+envs:
+    - name: "PROFILE"
+      value: "CACHING-SERVICE"
+ports:
+    - value: 11222
+run:
+    user: 185
+    cmd:
+        - "/opt/datagrid/bin/jdg-online-services.sh"
+modules:
+    repositories:
+        - path: modules
```

Comment:
This is optional. If modules are stored in "modules" directory next to image descriptor - it'll be picked up automatically.

Reply:
fixed